examples/react/css/src/common.css

```diff
@@ -0,0 +1,61 @@
+/* Motion reduction */
+@media (prefers-reduced-motion: reduce) {
+  [class^='ais-'] {
+    transition: none !important;
+  }
+}
+
+/* Focus outline */
+a[class^='ais-']:focus-visible,
+button[class^='ais-']:focus-visible,
+input[class^='ais-']:focus-visible {
+  outline: 2px solid
+    rgba(var(--ais-primary-color-rgb), var(--ais-primary-color-alpha));
+  outline-offset: 2px;
+}
+
+/* Widgets */
+[class^='ais-'][class*='-showMore'] {
+  border-radius: calc(var(--ais-spacing) / 4);
+  font-size: 0.9em;
+  font-weight: var(--ais-font-weight-medium);
+  color: #fff;
```

Comment:
should we have a variable here instead of defining `#fff`?

Reply:
I saw that pure white was used in the autocomplete classic theme without a variable, so I went that way.
But we could use something like `--ais-color-white-rgb: 255, 255, 255;`

Reply:
I feel like it should be a variable, just incase the background color is changed for example, white might not work etc.

Reply:
I agree, done!